magicc_sim_plugins/src/gazebo_ros_camera_utils.cpp

```diff
@@ -492,7 +494,7 @@ void GazeboRosCameraUtils::Init()
   else
   {
     // check against float precision
-    if (!gazebo::math::equal(this->focal_length_, computed_focal_length))
+    if (!ignition::math::equal(this->focal_length_, computed_focal_length))
```

Comment:
Does this work on all versions of gazebo?

Reply:
Thanks for catching that. I made the update.